applications/salk-portal/src/components/home/ExperimentList.jsx

```diff
@@ -263,7 +265,7 @@ const ExperimentList = (props) => {
       <Box p={5}>
         <Grid container item spacing={3}>
           {[1, 2, 3, 4, 5, 6, 7, 8, 9, 10].map((item, i) => (
-            <ExperimentCard i={`${heading}experiment_${i}`} tags={tags} heading={"Exploration of the spinal cord"} description={"Shared on Sept 2nd, 2021"} user={"name"} />
+            <ExperimentCard i={`${heading}experiment_${i}`} type={type} tags={tags} heading={"Exploration of the spinal cord"} description={type == "personal" ? "Sept 2nd, 2021": "Shared on Sept 2nd, 2021"} user={"name"} />
```

Comment:
why do you let the container decide what the heading and description is? This should be the responsibility of the card. 
also here I would pass in the experiment object and not all individual attributes. please also ask @vidhya-metacell what she thinks about this

Reply:
@paularah please create a dummy experiment object and pass here. Once you do that you can use the type to make the decision in the ExperimentCard rather than in the list. Also we are using the Experiment Card for communities as well which has in turn has different description and also an image on the corner. You would notice that decision is taken inside the the experiment card for community image display, in the same you can decide for the description in the card.

Reply:
@paularah @vidhya-metacell there is already an Experiment type in apiclient/workspaces/api.ts ;-)

or even better create a new models/experiment.ts and extend it from apiclient/workspaces/api.ts Experiment

Reply:
(it could be that you have to pull latest develop branch in your branch)

Reply:
@zoran would this be a class that implements the experiment intecace defined in apiclient/workspaces/api.ts?

Reply:
@paularah @zsinnema there is actually a third type of description being used for experiments in community tab.
I will them here for reference:
my experiments - just the date
shared - 'Shared on' + date
community - number of clones

Reply:
> @paularah @vidhya-metacell there is already an Experiment type in apiclient/workspaces/api.ts ;-)
> 
> or even better create a new models/experiment.ts and extend it from apiclient/workspaces/api.ts Experiment

@zsinnema maybe I am missing something, I can't find a type field on the experiment interface.

Reply:
of course there is no type on the experiments interface, I meant there is already an `Experiment` class (type)